embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/capture/startup/StartupServiceImpl.kt

```diff
@@ -1,12 +1,9 @@
 package io.embrace.android.embracesdk.internal.capture.startup
 
 import io.embrace.android.embracesdk.internal.spans.SpanService
-import io.embrace.android.embracesdk.internal.worker.BackgroundWorker
 
-internal class StartupServiceImpl(
-    private val spanService: SpanService,
-    @Suppress("UnusedPrivateMember")
-    private val backgroundWorker: BackgroundWorker
+class StartupServiceImpl(
```

Comment:
In the future, we can probably streamline and simplify this quite a bit and all its moments responsibilities are gone. Seems excess to have an entire service just providing startup info and creating a trace.